site/_posts/2013-11-20-inspectxml-dump-objects-as-xml.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: inspectXML – Dump objects as XML
 date: 2013-11-20
-tags: external operations cloudforms vmware rhev kvm cluster vm datastore XML
```

Comment:
The various "platformy" tags aren't useful because they have nothing to do with the article per se - that is someone looking at vmware, wouldn't care about this article.